frontend/packages/ux-editor/src/components/config/editModal/EditBooleanValue.tsx

```diff
@@ -32,18 +34,21 @@ export const EditBooleanValue = ({
 
   const getNewBooleanValue = () => !(component[propertyKey] ?? defaultValue);
 
+  const getHelpText = () => {
+    if (isValueExpression(component[propertyKey])) {
+      return t('ux_editor.component_properties.config_is_expression_message');
+    }
+    return componentPropertyHelpText(propertyKey) || helpText;
+  };
+
   return (
     <FormField
       id={component.id}
       value={component[propertyKey]}
       onChange={handleChange}
       propertyPath={component.propertyPath}
       componentType={component.type}
-      helpText={
-        isValueExpression(component[propertyKey])
-          ? t('ux_editor.component_properties.config_is_expression_message')
-          : componentPropertyHelpText(propertyKey)
-      }
+      helpText={getHelpText()}
```

Comment:
Nitpicking, do you think it might be simpler to read if making this a variable instead of a function? That way, others won’t need to look up the function definition and locate where it’s used. 🙂

```
const helpText = isValueExpression(component[propertyKey]) ? t('ux_editor.component_properties.config_is_expression_message') : componentPropertyHelpText(propertyKey)

```

Reply:
yes we can simplify it 👍